new-integration-tests/src/test/resources/wdt-models/wdt-singleclusterdomain-sampleapp-wls.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```

Comment:
is this file different from existing model file for single cluster domain?

Reply:
Yes. This is the difference:
AdminUserName: '@@PROP:AdminUserName@@'
AdminPassword: '@@PROP:AdminPassword@@'